app/services/xccdf_report_parser.rb

```diff
@@ -31,11 +31,11 @@ class UnknownRuleError < StandardError; end
   BENCHMARK_PREFIX = 'xccdf_org.ssgproject.content_benchmark_'
 
   def initialize(report_contents, message)
-    unless valid_message_format?(message)
-      raise MissingIdError, 'Missing Host id in message'
-    end
-
+    @id = message['id']
     @b64_identity = message['b64_identity']
```

Comment:
It's funny - this instance variable has been here for 3 years and has never been used. It's been `nil` the whole time! When @skateman was looking to use it with https://github.com/RedHatInsights/compliance-backend/pull/984/files#diff-8fb386345eced6ce59dafa10d303ccbf4acc6a6d4ad828d08758a0a7ff4f57e6R55, we both assumed it was there and non-nil.